ElectronicObserver/Window/Control/Paging/PagingControlViewModel.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+
+namespace ElectronicObserver.Window.Control.Paging;
+
+public partial class PagingControlViewModel : ObservableObject
+{
+	public int CurrentPage { get; set; } = 0;
+
+	private ObservableCollection<object?> Items { get; set; } = new();
+
+	public List<object?> ItemsPaged { get; set; } = new();
+
+	public int ItemsPerPage { get; set; } = 10;
+
+	public int LastPage => (int)Math.Ceiling(Items.Count / (decimal)ItemsPerPage);
+
+	public PagingControlTranslationViewModel PagingControl { get; set; } = new();
+
+	public PagingControlViewModel()
+	{
+		PropertyChanged += OnPagerUpdate;
+		PropertyChanged += OnPagerUpdate2;
+		UpdateCollection();
+	}
+
+	public PagingControlViewModel(ObservableCollection<object?> items) : this()
```

Comment:
Is there a need for this constructor?

Reply:
Was thinking we can reuse it one day in another view with an obervablecollection.
I'll just remove it